README.md

```diff
@@ -211,6 +211,22 @@ class InlineComponent < ViewComponent::Base
 end
 ```
 
+It is also possible to render variants inline by creating additional `call_` methods.
+
+```ruby
+class InlineVariantComponent < ViewComponent::Base
+  def call
+    link_to "Default", default_path
+  end
+
+  def call_phone
+    link_to "Phone", phone_path
+  end
+end
+```
+
+Using a mixture of templates and inline render methods in a component is supported, however if both exist for a component (or specific variant) the template will take priority.
```

Comment:
Reading this description, I wonder if it would be better to just raise an error in this case. I can't think of a case where we'd ever want to allow both to exist.